### ISSUE TITLE
sed -i  gets permission error

### ISSUE THREAD

**Issue by Ursula** [redhat-helloworld-msa/frontend#15]
Running Minishift with VirtualBox on Mac OSX

The frontend application is having an issue running the sed command since it is trying to write a temporary file to possibly the /tmp directory owned by root.   How do I get around this?  

In the Dockerfile for the frontend project on line 31 there is a  sed command

sed -i services.json \
        -e 's|OLACHAINURL|'"$OLACHAINURL"'|' \
        -e 's|HOLAURL|'"$HOLAURL"'|' \
        -e 's|BONJOURURL|'"$BONJOURURL"'|' \
        -e 's|ALOHAURL|'"$ALOHAURL"'|' \
        -e 's|OLAURL|'"$OLAURL"'|' \
        -e 's|APIGATEWAYURL|'"$APIGATEWAYURL"'|' \
    && sed -i index.html \
        -e 's|HYSTRIXDASHBOARDURL|'"$HYSTRIXDASHBOARDURL"'|' \
        -e 's|JAEGERQUERYURL|'"$JAEGERQUERYURL"'|' \
    && /bin/bash -c 'npm start'

When deployed to the container it gets this error:

sed: couldn't open temporary file ./sedq2WEJF: Permission denied

**Comment by rafabene** [comment]
I've updated the instructions at http://bit.ly/msainstructions

Since you're running minishift, don't forget to enable the anyuid addons: minishift addons enable anyuid

**Comment by Ursula** [comment]
Thank you.   Also I am getting a similar issue with  Hystrix. 

2018-03-05 23:02:47.430:WARN:oejs.HomeBaseWarning:main: This instance of Jetty is not running from a separate {jetty.base} directory, this is not recommended.  See documentation at http://www.eclipse.org/jetty/documentation/current/startup.html

2018-03-05 23:02:47.566:WARN:oejuc.FileNoticeLifeCycleListener:main:
java.io.FileNotFoundException: /opt/jetty/jetty.state (Permission denied)

How do I fix this error?
From the instructions I used 

$ oc process -f http://central.maven.org/maven2/io/fabric8/kubeflix/packages/kubeflix/1.0.17/kubeflix-1.0.17-kubernetes.yml | oc create -f -   

I noticed that there might be an issue in this version of Jetty that is used but when I tried to update to a new 1.0.28 yml file I receive this error from oc 
error: unable to parse "http://central.maven.org/maven2/io/fabric8/kubeflix/packages/kubeflix/1.0.28/kubeflix-1.0.28-kubernetes.yml", not a valid Template but *api.List
